tensorflow/compiler/xla/service/gpu/ir_emitter.h

```diff
@@ -183,6 +189,9 @@ class IrEmitter : public DfsHloVisitorWithDefault,
   // Hlo configuration data used during code generation.
   const HloModuleConfig& hlo_module_config_;
 
+  // GPU Target machine features for code generation
+  llvm_ir::LLVMTargetFeatures& llvm_target_features_;
```

Comment:
In case `IrEmitterUnnested` is constructed without an `llvm_target_features`, `llvm_target_features_` field in `IrEmitter` would become a garbage reference.